tests/test_cora_transform.py

```diff
@@ -432,15 +436,13 @@ def test_ukis_pdf(self):
                     self.assertFalse(any(a.startswith(i) for i in questions), "No answer in image.")
 
         self.assertIn("Respondent comment data.", pages[1])
+        os.remove(temp_testfile)
 
     @unittest.skip("Sample Generation")
     def test_ukis_zip(self):
         log = logging.getLogger("test")
-        tx = CORATransformer(log, self.settings, self.survey, self.data)
-        pdf = tx.create_formats(numberSeq=itertools.count())
-        tx.prepare_archive()
-        zipFile = tx.create_zip()
+        tx = CORATransformer(log, self.survey, self.data)
```

Comment:
Is this test being skipped?

Reply:
Its been skipped for ages , not a new change